docs/swupd.1.rst

```diff
@@ -266,6 +266,39 @@ SUBCOMMANDS
         Omit checking hash values. Instead only corrects missing files
         and directories and/or symlinks.
 
+    - `-Y, --picky`
+
+        List (without --fix) or remove (with --fix) files which should
+        not exist. Only files listed in the manifests should exist.
+
+    - `-X, --picky-tree=[PATH]`
+
+        Selects the sub-tree where --picky looks for extra files. To be
+        specified as absolute path. The default is `/usr`.
+
+    - `-w, --picky-whitelist=[RE]`
+
+        Any path matching the POSIX extended regular expression is
+        ignored by --picky. Matched directories get skipped
+        completely. Directory paths end in a slash. This makes it
+        possible to write expressions that match a directory but not
+        something else that happens to have the same base name.
+
+        The default is to ignore ``/usr/lib/kernel``,
+        ``/usr/lib/modules``, and ``/usr/local``.
+
+        Examples:
+
+        - ``^/var/|^/etc/machine-id$``
+
+            Ignores the ``/var`` directory and everything inside it
+            (because directories get skipped) and the
+            ``/etc/machine-id`` file.
```

Comment:
In this original example, the /var symlink would not be matched. If that had been desired, ``^/var$`` would have to be part of the regex.

I'll change it so that ``/var = ^(/var)$`` matches both the /var dir and /var symlink.